client/components/header/header.js

```
@@ -61,210 +61,127 @@ export default class Header extends React.Component {
-                        className="header-logo-image header-desktop-logo-image"
-                      />
-                    </Link>
-                  ) : null}
-                </div>
-              </div>
+      <a href={link.url} className={className} target="_blank" rel="noreferrer noopener" key={link.url}>
+        {getText(link.text, language)}
+      </a>
+    );
+  });
+};
```

ESLint

[error] Expected blank line between class members.

(lines-between-class-members)